migrations/20221226161124-create-designation.js

```diff
@@ -8,6 +8,13 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.UUID,
         defaultValue: Sequelize.literal("uuid_generate_v4()"),
+<<<<<<< HEAD
+      },
+      designation_code: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+=======
+>>>>>>> staging
```

Comment:
please review by yourself before adding code to git.
you have added conflict code in git